### PR TITLE
Feature: Enable PDO destination transactions by default

### PR DESCRIPTION
**Pull request by dextermb**
- Add disableTransactions method
- Return instance from ignoreIntegrityConstraintViolations method

### CODE REVIEW

**Comment by DivineOmega** [comment]
Thanks for this.

If you could fix the code coverage decrease, I'll get this merged in.

**Comment by DivineOmega** [comment]
Will deal with code coverage later.

File: docs/destinations/PDODestination.md [hunk 1]
```diff
@@ -44,7 +44,7 @@ $pdoDestination->ignoreIntegrityConstraintViolations();
 
 # Transactions
 
-By default PDO transactions are enabled. This helps speed up the migration process by keeping changes in memory until they are committed to disk. If you do not want to have transactions or is not supported you can easily disable this, as shown below.
+By default, database transactions are enabled. This helps speed up the migration process by keeping changes in memory until they are committed to disk. If you do not want to use transactions or they not supported by your database engine, you can easily disable them, as shown below.
```
Comment (dextermb) [inline docs/destinations/PDODestination.md]: > By default, database transactions are enabled.

Surely this should be specific as you might have a MYSQLi destination at some point?

Comment (DivineOmega) [reply docs/destinations/PDODestination.md]: It's specifically in the PDO Destination documentation, so I figured that was unnecessary.

Comment (dextermb) [reply docs/destinations/PDODestination.md]: Ah true, inside a specific file

Comment (DivineOmega) [reply docs/destinations/PDODestination.md]: :+1:

**Pull request merged**